hack/ginkgo-e2e.sh

```diff
@@ -89,7 +89,7 @@ fi
 export PATH=$(dirname "${e2e_test}"):"${PATH}"
 "${ginkgo}" "${ginkgo_args[@]:+${ginkgo_args[@]}}" "${e2e_test}" -- \
   "${auth_config[@]:+${auth_config[@]}}" \
-  --host="https://${KUBE_MASTER_IP-}" \
+  --host="https://${KUBE_MASTER_IP:-}" \
```

Comment:
It seems like the simplest thing is to keep it as is (and have it be `https://` when `KUBE_MASTER_IP` isn't set). That way we can assume that the value of the var begins with `https://` and don't have to add special casing. Then, we could consider doing a cleanup change in a future PR. Does this sound all right?